docs/docs/target/kubernetes.md

```diff
@@ -3,84 +3,115 @@
 !!! warning "EXPERIMENTAL"
     This feature might change without preserving backwards compatibility.
 
-## CLI
-The Trivy K8s CLI allows you to scan your Kubernetes cluster for 
+Trivy can connect to your Kubernetes cluster and scan it for security issues using the `trivy k8s` command. This page covers the technical capabilities of Trivy Kubernetes scanning.
+Trivy can also be installed *inside* your cluster as a Kubernetes Operator, and continuously scan it. For more about this, please see the [Trivy Operator][https://aquasecurity.github.io/trivy-operator/] project.
 
+When scanning a Kubernetes cluster, Trivy differentiates between the following:
+
+1. Cluster infrastructure (e.g api-server, kubelet, addons)
+1. Cluster configuration (e.g Roles, ClusterRoles). 
+1. Application workloads (e.g nginx, postgresql).
+
+When scanning any of the above, Trivy differentiates between the Kubernetes Resource definition (i.e the YAML that defines it), and the container image if relevant.
```

Comment:
Same here as in the other docs -- why is it the same under Target as in Coverage? is it necessary to duplicate the information or could we refer here to the Coverage docs above?

Reply:
the coverage section a high level overview of what things trivy can scan. the detailed info is in the relevant sections. I didn't feel like k8s bulbs belong under one of the existing coverage sections: OS/IaC so I added another section.